nautobot/core/views/utils.py

```diff
@@ -269,7 +270,7 @@ def prepare_cloned_fields(instance):
         for tag in instance.tags.all():
             params.append(("tags", tag.pk))
 
-    # Concatenate parameters into a URL query string
-    param_string = "&".join([f"{k}={v}" for k, v in params])
```

Comment:
Oof. I'm surprised this stuck around for as long as it has. Thanks for the fix!

If you'd like to go wild, there's some similarly sketchy-looking logic in `nautobot.core.tables.LinkedCountColumn.render()` that probably could do with a similar fix, though based on its context it's likely a lot less impactful. Not required for this PR!